### DISCUSSION TITLE
How do I switch Telegram profiles?

### DISCUSSION THREAD

**Discussion by Zik42** [d99kris/nchat#288] [Q&A]
I did register 2 telegram accounts via setup command. I dont see in docs on how to switch between them :(
Would be nice to have in help menu or in docs.

Thank you for suck great software you provide!

**Comment by d99kris** [comment]
Hi @Zik42 - to be able to run them separately you will need to set them up with separate config directories, otherwise they will all be loaded when starting nchat. Some references:
https://github.com/d99kris/nchat#5-how-to-use-telegram-and-whatsapp-concurrently
[Is there a way to toggle between Telegram and Whatsapp clients?](https://github.com/d99kris/nchat/discussions/269)
[Possible to run both Telegram and Whatsapp at the same time?](https://github.com/d99kris/nchat/discussions/258) 
If you already set up two telegram accounts with the same config dir, the simplest solution may be to simple delete the config dir `~/.config/nchat`, and set up two separate based on above links.

**Reply by Zik42** [reply]
Feel shame, that I did not see it main README. Thank you for support even so. It is a pleassure to use such a great software!

**Reply by d99kris** [reply]
No worries! Thanks for the positive feedback 👍
